src/components/Days/DayCard/DayCard.js

```diff
@@ -2,8 +2,15 @@ import React from 'react';
 import moment from 'moment';
 import logo from '../../../assets/sun.svg';
 import classes from './DayCard.module.scss';
+import { convertKelvinToCelsius } from '../../../utils/index';
 
 export const DayCard = (props) => {
+  const toggleKelvinCelsius = function () {
```

Comment:
The same logic is used in several components. Maybe we can move it to the util folder or something?